Patches/Poleepkwa Race/New_Prawns.xml

```diff
@@ -0,0 +1,189 @@
+<Patch>
+
+	<Operation Class="PatchOperationFindMod">
+    <mods>
+		<li>Poleepkwa Race Update</li>
+    </mods>
+		<match Class="PatchOperationSequence">
+		<operations>		
+
+		<li Class="PatchOperationAdd">
+		<xpath>Defs</xpath>
+		<value>
+
+  <!-- ========== Tribals ========== -->
+
+<PawnKindDef>
+    <defName>PpGrenadier</defName>
+    <race>Poleepkwa</race>
+    <label>boomer</label>
```

Comment:
They'll probably need packs, as their bulk is such they're not spawning with additional stick bombs.

Reply:
I don't think they need backpacks since vanilla tribal grenadiers do not always spawn with backpacks either.

Ingame they'll typically throw 1-2 grenades at most before dying, actually letting them carry all the stickbombs will just lead to every tribal raid giving you 30+ Free stickbombs.

Reply:
The ones I saw were spawning with only the single grenade they had equipped and no extras in their inventory. Whether or not this impacts every single one, I'm not positive.